website/ref/language.md

```diff
@@ -106,16 +106,75 @@ error.
 
 The three syntaxes above all evaluate to strings, and they are interchangeable.
 For instance, `xyz`, `'xyz'` and `"xyz"` are different syntaxes for the same
-string, and they are always equivalent.
+string, and they are always equivalent with the exception of
+**escape sequences** as documented above.
 
-Elvish does have a dedicated `float64` number type, but it does not have a
-literal syntax; instead it needs to be constructed using a function call, as in
-`put (float64 42)`; the argument `42` is a string. For convenience, all Elvish
-functions that expect number arguments also accept strings that can be parsed as
-a number, so explicit number constructors are not needed often.
+# Numeric data type
 
+Elvish has a numeric data type, `float64`, created using this syntax:
+`(float64 numeric-literal)`. For convenience, all Elvish functions that
```

Comment:
It's inaccurate to call the argument `numeric-literal`. The concept of "literal" is syntactical: this is implying that the argument must be something *literal*, as opposed to, say, a variable.

I prefer using a variable as a placeholder here, so just say `(float64 $number)`.